sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -271,6 +359,10 @@ def from_connection_string(
          For detailed partition context information, please refer to
          :class:`PartitionContext<azure.eventhub.aio.PartitionContext>`.
         :type on_event: Callable[~azure.eventhub.aio.PartitionContext, ~azure.eventhub.EventData]
+        :keyword float max_wait_time: `on_event` is called with its param `event` being `None`
+         if `max_wait_time` is not None nor 0 and if no event has been received after `max_wait_time` second(s)
+         since `on_event` is last called.
+         Default is None.
```

Comment:
A bit confused by the description, I would expect more explicit here:

If it's None or 0, the callback won't be called.
If it's specified, it will be called with `event` being None if no event is received within `max_wait_time` since `on_event` is last called.